data/korath ships.txt

```diff
@@ -17,12 +17,11 @@ ship "Korath Raider"
 		"hull" 9000
 		"required crew" 145
 		"bunks" 250
-		"mass" 720
+		"mass" 650
 		"drag" 12
 		"heat dissipation" .5
-		"fuel capacity" 600
```

Comment:
The ships should have some base fuel capacity to support at least a single jump with their default drive.

Reply:
Each Fuel Collector allows a single jump drive jump or two hyperdrive jumps, and each Raider and World-Ship variant is equipped with several of them.

Reply:
And if the player plunders that outfit and then captures the ship, it is unable to jump *at all*
Note that the player will likely be unaware of the intricacies of the outfit prior to obtaining one. The name "Fuel Collector" does not imply that it also stores fuel, after all.

Reply:
Similarily, if the player plunders the jump drive prior to capturing the ship, it won't be able to jump at all either. Anyway, I've now integrated one fuel collector (i.e. 200 capacity + .5 ramscoop) fixed in each Raider and World-Ship.